<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>997</width>
    <height>662</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>StereoView Test</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>420</y>
      <width>511</width>
      <height>16</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>WaitCursor</cursorShape>
    </property>
    <property name="value">
     <number>100</number>
    </property>
   </widget>
   <widget class="StereoGraphicsView" name="imagesView">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>981</width>
      <height>401</height>
     </rect>
    </property>
    <property name="mouseTracking">
     <bool>true</bool>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="dragMode">
     <enum>QGraphicsView::ScrollHandDrag</enum>
    </property>
    <property name="transformationAnchor">
     <enum>QGraphicsView::AnchorUnderMouse</enum>
    </property>
    <property name="resizeAnchor">
     <enum>QGraphicsView::AnchorUnderMouse</enum>
    </property>
    <property name="viewportUpdateMode">
     <enum>QGraphicsView::SmartViewportUpdate</enum>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="console">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>440</y>
      <width>511</width>
      <height>181</height>
     </rect>
    </property>
    <property name="undoRedoEnabled">
     <bool>false</bool>
    </property>
    <property name="readOnly">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QGroupBox" name="pmvsGroup">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>420</y>
      <width>301</width>
      <height>101</height>
     </rect>
    </property>
    <property name="title">
     <string>PMVS2 Options</string>
    </property>
    <widget class="QSpinBox" name="levelBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="value">
      <number>0</number>
     </property>
    </widget>
    <widget class="QLabel" name="levelLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>20</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Level</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="thresholdLabel">
     <property name="geometry">
      <rect>
       <x>170</x>
       <y>20</y>
       <width>51</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Threshold</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="wsizeBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>9</number>
     </property>
    </widget>
    <widget class="QLabel" name="wsizeLabel">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>60</y>
       <width>21</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Win Size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="maxAngleLabel">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>60</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Angle</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="maxAngleBox">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>60</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>360</number>
     </property>
     <property name="value">
      <number>5</number>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="thresholdDoubleBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.800000000000000</double>
     </property>
    </widget>
    <widget class="QSpinBox" name="csizeBox">
     <property name="geometry">
      <rect>
       <x>120</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="value">
      <number>2</number>
     </property>
    </widget>
    <widget class="QLabel" name="csizeLabel">
     <property name="geometry">
      <rect>
       <x>80</x>
       <y>20</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Cell Size</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="quadDoubleBox">
     <property name="geometry">
      <rect>
       <x>230</x>
       <y>60</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>100.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>2.500000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="quadLabel">
     <property name="geometry">
      <rect>
       <x>190</x>
       <y>60</y>
       <width>31</width>
       <height>20</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Quad</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="surfGroup">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>520</y>
      <width>161</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>SURF Options</string>
    </property>
    <widget class="QSpinBox" name="hessianBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>20</y>
       <width>42</width>
       <height>22</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>true</bool>
     </property>
     <property name="maximum">
      <number>1000</number>
     </property>
     <property name="singleStep">
      <number>10</number>
     </property>
     <property name="value">
      <number>300</number>
     </property>
    </widget>
    <widget class="QLabel" name="hessianLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>46</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Hessian</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="responseLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>21</height>
      </rect>
     </property>
     <property name="text">
      <string>Response</string>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="responseDoubleBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>0.000001000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.001000000000000</double>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="groupBox">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>420</y>
      <width>151</width>
      <height>171</height>
     </rect>
    </property>
    <property name="title">
     <string>Kmatcher Options</string>
    </property>
    <widget class="QSpinBox" name="neighboursBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>20</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>20</number>
     </property>
     <property name="value">
      <number>1</number>
     </property>
    </widget>
    <widget class="QLabel" name="neighboursLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>20</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Neighbours</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="maxItersBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>50</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="accelerated">
      <bool>false</bool>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>100</number>
     </property>
     <property name="value">
      <number>10</number>
     </property>
    </widget>
    <widget class="QLabel" name="maxItersLabel">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>50</y>
       <width>61</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Max Iters</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="avgPayoffDoubleBox">
     <property name="geometry">
      <rect>
       <x>90</x>
       <y>80</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>2</number>
     </property>
     <property name="minimum">
      <double>0.050000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.050000000000000</double>
     </property>
     <property name="value">
      <double>0.600000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="avgPayoffLabel">
     <property name="geometry">
      <rect>
       <x>40</x>
       <y>70</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Avg Payoff</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QLabel" name="payoffAlphaLabel">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>100</y>
       <width>41</width>
       <height>31</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Payoff Alpha</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="payoffAlphaDoubleBox">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>110</y>
       <width>81</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>7</number>
     </property>
     <property name="minimum">
      <double>0.000001000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.001000000000000</double>
     </property>
     <property name="value">
      <double>0.001000000000000</double>
     </property>
    </widget>
    <widget class="QDoubleSpinBox" name="qualityDoubleBox">
     <property name="geometry">
      <rect>
       <x>100</x>
       <y>140</y>
       <width>41</width>
       <height>22</height>
      </rect>
     </property>
     <property name="decimals">
      <number>1</number>
     </property>
     <property name="minimum">
      <double>0.100000000000000</double>
     </property>
     <property name="maximum">
      <double>1.000000000000000</double>
     </property>
     <property name="singleStep">
      <double>0.100000000000000</double>
     </property>
     <property name="value">
      <double>0.700000000000000</double>
     </property>
    </widget>
    <widget class="QLabel" name="qualityLabel">
     <property name="geometry">
      <rect>
       <x>50</x>
       <y>140</y>
       <width>41</width>
       <height>21</height>
      </rect>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="text">
      <string>Quality</string>
     </property>
     <property name="alignment">
      <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
     </property>
     <property name="wordWrap">
      <bool>true</bool>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="viewerBox">
    <property name="geometry">
     <rect>
      <x>860</x>
      <y>520</y>
      <width>131</width>
      <height>81</height>
     </rect>
    </property>
    <property name="title">
     <string>Viewer Options</string>
    </property>
    <widget class="QCheckBox" name="rescaleCheckBox">
     <property name="enabled">
      <bool>true</bool>
     </property>
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>20</y>
       <width>72</width>
       <height>18</height>
      </rect>
     </property>
     <property name="text">
      <string>Rescale</string>
     </property>
     <property name="checked">
      <bool>false</bool>
     </property>
    </widget>
    <widget class="QSpinBox" name="wBox">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>640</number>
     </property>
    </widget>
    <widget class="QSpinBox" name="hBox">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>50</y>
       <width>51</width>
       <height>22</height>
      </rect>
     </property>
     <property name="minimum">
      <number>1</number>
     </property>
     <property name="maximum">
      <number>10000</number>
     </property>
     <property name="value">
      <number>480</number>
     </property>
    </widget>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>997</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionOpenDir"/>
    <addaction name="actionOpenFiles"/>
    <addaction name="separator"/>
    <addaction name="actionClear"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuStereoView">
    <property name="title">
     <string>StereoView</string>
    </property>
    <widget class="QMenu" name="menuFeature_Detection">
     <property name="title">
      <string>Detect Features</string>
     </property>
     <addaction name="actionSIFT_Keypoints"/>
     <addaction name="actionSIFTpp_Keypoints"/>
     <addaction name="actionOpenSURF_Keypoints"/>
     <addaction name="actionOpenCV_SURF_Keypoints"/>
    </widget>
    <widget class="QMenu" name="menuRecon_Scene">
     <property name="title">
      <string>3D Scene</string>
     </property>
     <addaction name="actionPMVS2"/>
    </widget>
    <widget class="QMenu" name="menuBundler">
     <property name="title">
      <string>Bundle Adjustment</string>
     </property>
     <addaction name="actionBundler"/>
     <addaction name="actionAsync_Bundler"/>
    </widget>
    <widget class="QMenu" name="menuTracking">
     <property name="title">
      <string>Track Correspondences</string>
     </property>
     <addaction name="actionKmatcher_Single_Shot"/>
     <addaction name="actionKmatcher_Concurrent"/>
     <addaction name="actionBundler_Keymatcher"/>
    </widget>
    <addaction name="menuFeature_Detection"/>
    <addaction name="menuTracking"/>
    <addaction name="separator"/>
    <addaction name="menuBundler"/>
    <addaction name="menuRecon_Scene"/>
    <addaction name="separator"/>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
    <addaction name="actionStatusBarLogEnabled"/>
   </widget>
   <widget class="QMenu" name="menuTest">
    <property name="title">
     <string>Test</string>
    </property>
    <addaction name="actionMulti_threaded_Progress_Bar"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuStereoView"/>
   <addaction name="menuOptions"/>
   <addaction name="menuTest"/>
  </widget>
  <widget class="QStatusBar" name="statusBar">
   <property name="toolTip">
    <string/>
   </property>
   <property name="accessibleName">
    <string/>
   </property>
   <property name="accessibleDescription">
    <string/>
   </property>
  </widget>
  <action name="actionOpenDir">
   <property name="text">
    <string>Open directory..</string>
   </property>
   <property name="toolTip">
    <string>Open image files from directory</string>
   </property>
  </action>
  <action name="actionStatusBarLogEnabled">
   <property name="checkable">
    <bool>true</bool>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
   <property name="text">
    <string>StatusBar Log Enabled</string>
   </property>
   <property name="toolTip">
    <string>Toggle StatusBar logger</string>
   </property>
  </action>
  <action name="actionOpenFiles">
   <property name="text">
    <string>Open image files..</string>
   </property>
   <property name="toolTip">
    <string>Open one or more image files</string>
   </property>
  </action>
  <action name="actionClear">
   <property name="text">
    <string>Clear image buffer</string>
   </property>
  </action>
  <action name="actionKmatcher_Single_Shot">
   <property name="text">
    <string>Kmatcher Single Shot</string>
   </property>
  </action>
  <action name="actionPMVS2">
   <property name="text">
    <string>PMVS2</string>
   </property>
  </action>
  <action name="actionSIFT_Keypoints">
   <property name="text">
    <string>SIFT Keypoints</string>
   </property>
  </action>
  <action name="actionBundler">
   <property name="text">
    <string>Bundler</string>
   </property>
  </action>
  <action name="actionOpenSURF_Keypoints">
   <property name="text">
    <string>OpenSURF Keypoints</string>
   </property>
  </action>
  <action name="actionOpen_PLY_file">
   <property name="text">
    <string>Open PLY file..</string>
   </property>
  </action>
  <action name="actionFull_Recon">
   <property name="text">
    <string>Full Recon</string>
   </property>
  </action>
  <action name="actionAsync_Bundler">
   <property name="text">
    <string>Async Bundler</string>
   </property>
  </action>
  <action name="actionMulti_threaded_Progress_Bar">
   <property name="text">
    <string>Multi-threaded Progress Bar</string>
   </property>
  </action>
  <action name="actionOpenCV_SURF_Keypoints">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>OpenCV SURF Keypoints</string>
   </property>
  </action>
  <action name="actionKmatcher_Concurrent">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Kmatcher Concurrent</string>
   </property>
  </action>
  <action name="actionBundler_Keymatcher">
   <property name="text">
    <string>Bundler Keymatcher</string>
   </property>
  </action>
  <action name="actionSIFTpp_Keypoints">
   <property name="text">
    <string>SIFT++ Keypoints</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>StereoGraphicsView</class>
   <extends>QGraphicsView</extends>
   <header>stereographicsview.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
